### PR TITLE
Rev/202107

### PR DESCRIPTION
**Pull request by gjbex**
Revision of the documentation:
* updates for new versions/policies
* format cleanup
* marker disambiguation.

### CODE REVIEW

**Review by smoors — APPROVED** [review]
LGTM

**Comment by sbecuwe** [comment]
Thanks for the in depth review of the documentation!

File: source/access/nx_start_guide.rst [hunk 1]
```diff
@@ -6,92 +6,80 @@ NX start guide
...
 
-   If you experience problems with connection please switch to Protocol
-   NX and port 4000.
+            .. note::
 
--  These login nodes cannot be used to access the cluster from the terminal.   
+                These login nodes cannot be used to access the cluster from the terminal.   
 
-Choose the authentication **Use the system login.**
+#. In the **Configuration** pane,
```
Comment (mselwa) [inline source/access/nx_start_guide.rst:36]: pane, or panel. Nort sure about the ,

**Review by mselwa — APPROVED** [review]
Thank you.

**Pull request merged**